File: C/PRAK605-2210817220029-SitiAinurRahmawati.c
#include <stdio.h>
int main () {
    int A [50][50], B [50][50], C [50][50];
    int a, b, c, ordo;
    scanf ("%d", &ordo);
    printf("Matriks A\n");
    for (a=0;a<ordo;a++) {
        for (b=0;b<ordo;b++) {
            scanf("%d", &A [a][b]);}}
    printf ("Matriks B\n");
    for (a=0;a<ordo;a++) {
        for (b=0;b<ordo;b++) {
            scanf("%d", &B [a][b]);}}
    for (a=0;a<ordo;a++) {
        for (b=0;b<ordo;b++) {
            int jumlah=0;
            for (c=0;c<ordo;c++) {
                jumlah = jumlah + A [a][c]*B[c][b];}
            C[a][b]=jumlah;}}
    printf("Matriks AXB\n");
    for (a=0;a<ordo;a++) {
        for (b=0;b<ordo;b++) {
            printf("%d ", C [a][b]);}
        printf("\n");}
}

File: PYTHON/PRAK605-2210817220029-SitiAinurRahmawati.py
ordo = int(input())
matriksA = []
print ("Matriks A")
for a in range (ordo) :
    elemen = list(map(int, input().split()))
    matriksA.append(elemen)
matriksB = []
print ("Matriks B")
for a in range (ordo) :
    elemen = list(map(int, input().split()))
    matriksB.append(elemen)
print("Matriks AXB")
hasil = []
for a in range (ordo) :
    hasil.append([])
    for b in range (ordo) :
        jumlah = 0
        for c in range (ordo) :
            jumlah = jumlah + matriksA [a][c] * matriksB [c][b]
        hasil[a].append(jumlah)
for baris in hasil:
    for elemen in baris:
        print(elemen, end=" ")
    print()